engine/univMinim.ml

```diff
@@ -273,12 +273,13 @@ let normalize_context_set ~lbound g ctx us algs weak =
   let (ctx, csts) = ContextSet.levels ctx, ContextSet.constraints ctx in
   (* Keep the Prop/Set <= i constraints separate for minimization *)
   let smallles, csts =
-    Constraint.partition (fun (l,d,r) -> d == Le && (Level.equal l lbound || Level.is_sprop l)) csts
+    Constraint.partition (fun (l,d,r) -> d == Le && (Level.equal l lbound || Level.is_prop l || Level.is_sprop l)) csts
```

Comment:
The lbound use here was incorrect.

Reply:
@mattam82 maybe we shouldn't use lbound and instead just separate based on is_small (as it used to be)?

Reply:
That sounds right indeed.